DIY Peach (Ready) (#34)
scuttlebutt-eu/important-documents: DIY-Peach!.md

Guide for growing a Peach
====

> As inspired by [Patterns for Decentralized Organizing](https://docs.wixstatic.com/ugd/8246de_248a47ddbd3c4d209667ff5508ee300b.pdf), [IDOARRT](https://toolbox.hyperisland.com/idoarrt-meeting-design), [9 Artifacts To Seed a Project Team](http://kfjournal.org/wp/index.php/2015/10/19/9-artifacts-to-seed-a-project-team/) and [Pods](https://handbook.enspiral.com/guides/pods.html) and [Stewardship](https://loomio.coop/stewarding.html) from Enspiral. 

Reviewer: fyi i think this version of  Patterns for Decentralized Organizing is just a shorter pamphlet based on the longer book

> This is a proposal for how you can [grow a peach](https://github.com/scuttlebutt-eu/important-documents/pull/33). A peach is a form of working group which together built up the European Scuttlebutt Consortium tree. Feel free to hack!  

Reviewer 2: Before merging this PR, I don't think we should link to other PRs. Documents should link to other documents, and PRs aren't such.

> This is a proposal for how you can [grow a peach](https://github.com/scuttlebutt-eu/important-documents/pull/33). A peach is a form of working group which together built up the European Scuttlebutt Consortium tree. Feel free to hack!  

Reviewer: maybe remove this link, doesn't add a lot of context

> This is a proposal for how you can grow a Peach. A Peach is a form of working group which together built up the European Scuttlebutt Consortium tree. Feel free to hack!

Reviewer 3: ```suggestion
This is a proposal for how you can grow a Peach. A Peach is a kind of working group and method that smaller groups can use to get stuff done within the Consortium. Feel free to deviate from this guide when creating your own Peach.
```

Reviewer 2: y

> 1. **Compassion Building** - Make time to get to know each other (more)! Some ways to do this: Tell your life story to each other through life charts of highs and lows. Have meetings where you talk about other things than what you usually work with. Distribute care labour and strengthen the teams collective EQ. Set up a peer-to-peer support system like Stewardship.

Reviewer 2: What's EQ?
What's Stewardship?
Perhaps have links for more info?

Reviewer: stewardship is defined in one of the documents linked above

Reviewer: yeah i think EQ could be replaced

> 1. **Compassion Building** - Make time to get to know each other (more)! Some ways to do this: Tell your life story to each other through life charts of highs and lows. Have meetings where you talk about other things than what you usually work with. Distribute care labour and strengthen the team's collective emotional intelligence. Set up a peer-to-peer support system like Stewardship (defined in one of the links above).

Reviewer 3: ```suggestion
 1. **Compassion Building** - Make time to get to know each other (more)! Some ways to do this: Tell your life story to each other through sharing highs and lows, for example using life charts. Have meetings where you talk about topics other than what you usually work on. Distribute care labour and strengthen the team's collective emotional intelligence. Set up a peer-to-peer support system like Stewardship (defined in one of the links above).
```

Reviewer 2: y

> 2. **Interaction Format** - Talk about and save the conclusions for the future:

Reviewer 3: ```suggestion
 2. **Interaction Format** - Talk about and document the conclusions for the future:
```

Reviewer 2: y

> 	1. Set an intention and desired ourcome for your Peach.

Reviewer 3: ```suggestion
	1. Set an intention and desired outcome for your Peach.
```

Reviewer 2: yyyyyy

	2. Communicate expectations. How do you plan to contribute? How much time and with what kind of work scope? Be as specific as possible to help avoid missunderstandings or boundary crossings. 

> 	3. Decide how you will work together / when. Async/Meeting dates/Schedules etc.

Reviewer 3: ```suggestion
	3. Decide how you will work together / when. Async/meeting dates/schedules etc.
```

Reviewer 2: y

	4. Decide how you will communicate. Where will you communicate? What are the expectations of response time?
	5. Action Item Tracking System - How do we track what needs to be done?

3. **Culture**

> 	1. Rituals - are practices that help boost the interactivity of your calls. It could be as simple as holding a minute of silence at the start of the call.

Reviewer 3: ```suggestion
	1. Rituals - Practices that help boost the interactivity of your calls. It could be as simple as holding a minute of silence at the start of the call.
```

Reviewer 2: yyy
